scripts/perplexity.py

```diff
@@ -20,6 +20,7 @@
 import math
 import os
 import pathlib
+import psutil
```

Comment:
Do we need to add `psutil` to `scripts/requirements.txt`?

Reply:
No, it's already installed as a dependency of one of the packages in requirements.txt